linux-cachyos-bore/PKGBUILD

```diff
@@ -10,9 +10,7 @@
 _cachy_config=${_cachy_config-y}
 
 ### Selecting the CPU scheduler
-# ATTENTION - one of six predefined values should be selected!
-# 'bmq' - select 'BitMap Queue CPU scheduler'
-# 'pds' - select 'Priority and Deadline based Skip list multiple queue CPU scheduler'
+# ATTENTION - one of eight predefined values should be selected!
```

Comment:
Here I think the description needs to be changed (I mean the number eight)

Reply:
Do you want to replace the comment in general or?

Reply:
Yes, let's adapt the description to the current situation.